compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/ConvertConv2DToWinograd.cpp

```diff
@@ -59,7 +52,10 @@ static bool hasValidStridesAndDilations(Operation *op) {
   return true;
 }
 
-static bool isValidConv2d(Operation *op, bool &isNchw) {
+static bool isValidConv2d(Operation *op, bool &isNchw, bool ignoreAnnotations) {
+  if (!ignoreAnnotations && !op->hasAttr(kWinogradAttr)) {
```

Comment:
I think having this here is confusing. What are you trying to check. I am basically missing what `ignoreAnnotations` does

Reply:
I changed the option to `replace-all-convs`. Is that a better name?